src/Gumps/GumpToolbar.cpp

```diff
@@ -0,0 +1,174 @@
+//
+//  GumpToolbar.cpp
+//  crossuo
+//
+//  Created by Jean-Martin Miljours on 19-12-21.
+//
+
```

Comment:
Put this instead, we need to start getting these standardized as we create new files:
```
// AGPLv3 License
// Copyright (c) 2020 Jean-Martin Miljours
// Copyright (c) 2020 CrossUO Team
```

Reply:
done